AzerothCore Commit 4d349e8 causes compile failure due to AH rework

https://github.com/azerothcore/azerothcore-wotlk/commit/4d349e87b53b4921568163d8189369a62ab2d0f5

PlayerMethods.h:2483:61: fatal error: no member named 'GetAuctionHouseEntry' in 'AuctionHouseMgr'

```
In file included from /home/???/azerothcore/modules/mod-eluna/src/LuaEngine/LuaFunctions.cpp:24:
/home/???/azerothcore/modules/mod-eluna/src/LuaEngine/PlayerMethods.h:2483:61: fatal error: no member named 'GetAuctionHouseEntry' in 'AuctionHouseMgr'
 2483 |         AuctionHouseEntry const* ahEntry = AuctionHouseMgr::GetAuctionHouseEntry(unit->GetFaction());
      |                                            ~~~~~~~~~~~~~~~~~^
1 error generated.
```

--- a/src/LuaEngine/PlayerMethods.h
+++ b/src/LuaEngine/PlayerMethods.h
@@ -2479,10 +2479,12 @@ namespace LuaPlayer
     {
         Unit* unit = Eluna::CHECKOBJ<Unit>(L, 2);
 
-#if defined TRINITY || AZEROTHCORE
-        AuctionHouseEntry const* ahEntry = AuctionHouseMgr::GetAuctionHouseEntry(unit->GetFaction());
+#if defined AZEROTHCORE
+    AuctionHouseEntry const* ahEntry = AuctionHouseMgr::GetAuctionHouseEntryFromFactionTemplate(unit->GetFaction());
+#elif defined TRINITY
+    AuctionHouseEntry const* ahEntry = AuctionHouseMgr::GetAuctionHouseEntry(unit->GetFaction());
 #else
-        AuctionHouseEntry const* ahEntry = AuctionHouseMgr::GetAuctionHouseEntry(unit);
+    AuctionHouseEntry const* ahEntry = AuctionHouseMgr::GetAuctionHouseEntry(unit);
 #endif
         if (!ahEntry)
             return 0;

